docs/modules/clusters/pages/ucn-member-side.adoc

```diff
@@ -0,0 +1,89 @@
+= Member Configuration
+:description: {ucn} are referenced by name, and can be statically configured in the Hazelcast configuration file. You can specify JAR and JARs In ZIP resources in the configuration file.
+:page-enterprise: true
+:page-beta: true
+
+{description}
+
+To configure {ucn} using the _hazelcast_ configuration file, in either XML or YAML format, complete the following steps:
+
+. If not already open, open the _hazelcast_ configuration file in your preferred editor and scroll to the {ucn} section
+. If required, add a `java-serialization-filter` to verify classes when loaded into the class loader during namespace deployment
++
+For further information on the configuration of the `java-serialization-filter`, see the xref:serialization:serialization-configuration.adoc[] topic.
+
+. Define the name of the namespace under the line in which you enabled {ucn}, as follows:
++
+[tabs]
+====
+XML::
++
+[source,xml]
+----
+<user-code-namespaces enabled="true">
+    :
+    <user-code-namespace name="myNameSpace">
+    </user-code-namespace>
+</user-code-namespaces>
+----
+
+YAML::
++
+[source,yaml]
+----
+user-code-namespaces:
+   enabled: true
+   :
+   myNameSpace:
+----
+====  
+
+. Define the resources
++
+To define a resource, you must provide the resource type, and the location (URL) in which the resource can be found. You can also provide an identifier for the resource. If you do not provide an identifier, the specified URL is used.
++
+You can define multiple {ucn}, if required, by repeating steps 3 and 4 for your other namespaces.
++
+The resource type can be either JAR (`jar`) or JARs in ZIP (`jars-in-zip`); the following extract provides examples for both of these resource types:
++
+[tabs]
+====
+XML::
++
+[source,xml]
+----
+<user-code-namespaces enabled="true">
+    :
+    <user-code-namespace name="myNameSpace">
+        <jar id="jarID">
+            <url>file:./src/class/usercodedeployment/MyJarFile.jar</url>
+        </jar>
+        <jars-in-zip id="MyJARsInZIPFileID">
+            <url>file:./src/class/usercodedeployment/MyJarZIP.jar</url>
+        </jars-in-zip>
+    </user-code-namespace>
+</user-code-namespaces>
+----
+
+YAML::
++
+[source,yaml]
+----
+user-code-namespaces:
+   enabled: true
+   :
+   myNameSpace:
+    - id: jarID
+      url: "file:./src/class/usercodedeployment/MyJarFile.jar"
+    - id: MyJARsInZIPFileID
+      url: "file:./src/class/usercodedeployment/MyJarZIP.jar"
```

Comment:
We should use absolute root paths for our config examples, as that's the standard practice we advocate for elsewhere. Example: `file:///etc/hazelcast/your-configuration-file-1.xml`